### PR TITLE
Remove slurm-startup script v5 example and references

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: docs/vm-images.md
Comment (nick-stroud) [inline docs/vm-images.md:1]: We can't just delete this row, let's discuss.

Comment (harshthakkar01) [reply docs/vm-images.md:1]: Updated.

File: docs/vm-images.md [hunk 1]
```diff
@@ -207,10 +207,10 @@ description of our support for Windows images.
 <tr>
   <th>Startup script</th>
   <th></th>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
+  <td><a href="../community/examples/hpc-slurm-ramble-gromacs.yaml">✓</a></td>
+  <td>✓</td>
+  <td>✓</td>
```
Comment (nick-stroud) [inline docs/vm-images.md:212]: Same here as #2327 
We will want to update this once we migrate `tools/validate_configs/test_configs/slurm-gcp-v5-startup-scripts.yaml`

**Review by nick-stroud — APPROVED** [review]

File: docs/vm-images.md (hunk 1 above)
Comment (harshthakkar01) [reply docs/vm-images.md:212]: Added this PR as well in the canonical bug assigned to me. Will update the doc once we have v6 examples.

**Pull request merged**